README.md

```diff
@@ -1358,6 +1358,8 @@ See also [Are we game yet?](https://arewegameyet.rs)
   * [mozilla/neqo](https://github.com/mozilla/neqo) — an Implementation of QUIC written in Rust
 * P2P
   * [libp2p/rust-libp2p](https://github.com/libp2p/rust-libp2p) — The Rust Implementation of libp2p networking stack. [![Circle CI](https://circleci.com/gh/libp2p/rust-libp2p.svg?style=svg)](https://circleci.com/gh/libp2p/rust-libp2p)
+* NATS
+  * [nats-io/nats.rs](https://github.com/nats-io/nats.rs) — Rust client for NATS, the cloud native messaging system.
```

Comment:
I'm seeing a Github Actions build there. Can you please add the build badge to this as per https://github.com/rust-unofficial/awesome-rust/blob/master/CONTRIBUTING.md ?

Reply:
Done.